packages/account/src/Components/trading-assessment/trading-assessment-dropdown.tsx

```diff
@@ -88,26 +91,27 @@ const TradingAssessmentDropdown = ({
                                         disabled={disabled_items.includes(question.form_control)}
                                         error={meta.touched && meta.error}
                                     />
-                                </DesktopWrapper>
-                                <MobileWrapper>
-                                    <Text as='h1' color='prominent' weight='bold' size='xs'>
-                                        {question?.question_text}
-                                    </Text>
-                                    <SelectNative
-                                        {...field}
-                                        placeholder={localize('Please select')}
-                                        label={localize('Please select')}
-                                        name={question?.form_control}
-                                        list_items={question?.answer_options}
-                                        onChange={e => {
-                                            onChange(e, question.form_control, setFieldValue);
-                                        }}
-                                        value={values[question.form_control]}
-                                        hide_top_placeholder
-                                        disabled={disabled_items.includes(question.form_control)}
-                                        error={meta.touched && meta.error}
-                                    />
-                                </MobileWrapper>
+                                ) : (
+                                    <div>
```

Comment:
is it expected to add here additional div?
no impact on styles?

Reply:
yes as its needed to wrap <text> and <selectnative>

Reply:
@Dhruv-deriv , but MobileWrapper and DesktopWrapper contained just Fragment, and we add div instead, why do we need it? we add additional element just to wrap, and it can impact styling.
if we need just remove DesktopWrapper and MobileWrapper and keep some elements wrapped, should we use Fragment instead of div?
cc: @amina-deriv

Reply:
When we removed DesktopWrapper and MobileWrapper and simply relied on using Fragment to group our elements, we encountered an error that we need to put the content in a parent container(<div> ).  therefore, we had to introduced a (a <div>) to wrap these elements properly. 

according to me as `<DesktopWrapper>......content....</DesktopWrappe> `similarly `<MobileWrapper> `were parent containers but after removing them the content didnt get wrapped anywhere thats why in order to wrap it we put a div 

i will check again and provide the screen shot as well

Reply:
alright, correct, but DesktopWrapper and MobileWrapper is just a Fragments, so why we changed Fragments to divs?

Reply:
alright will make the change from div to fragment